openapi.yaml

```
@@ -15,7 +15,7 @@ servers:
     url: "http://localhost:5002"
 tags:
   - name: Session Verification
-    description: Not supported endpoints. Use with caution.
+    description: Endpoints behind CORS validation. Unable to use them from outside the domain `hashscan.io`.
```

Say the deployed domain e.g/ example.io
```suggestion
    description: Endpoints behind CORS validation. Unable to use them from outside the deployed domain e.g `example.io`.
```